Sources/SwiftFormat/PrettyPrint/PrettyPrintBuffer.swift

```diff
@@ -86,7 +89,15 @@ struct PrettyPrintBuffer {
     }
 
     guard numberToPrint > 0 else { return }
-    writeRaw(String(repeating: "\n", count: numberToPrint))
+    for number in 0..<numberToPrint {
+      if shouldIndentBlankLines, number >= 1 {
+        writeRaw(currentIndentation.indentation())
+        writeRaw("\n")
+      } else {
+        writeRaw("\n")
+      }
```

Comment:
```suggestion
      if shouldIndentBlankLines, number >= 1 {
        writeRaw(currentIndentation.indentation())
      }
      writeRaw("\n")
```

Would be slightly simpler